dev-docs/adunit-reference.md

```diff
@@ -42,6 +42,7 @@ See the table below for the list of properties on the ad unit. For example ad un
 | `renderer` | Optional | Object | Custom renderer, typically used for [outstream video](/dev-docs/show-outstream-video-ads.html) |
 | `video` | Optional | Object | Used to link an Ad Unit to the [Video Module][videoModule]. For allowed params see the [adUnit.video reference](#adunitvideo). |
 | `deferBilling` | Optional | Boolean | Used by a publisher to flag adUnits as being separately billable. This allows for a publisher to trigger billing manually for winning bids. See [pbjs.triggerBilling](/dev-docs/publisher-api-reference/triggerBilling.html) and [onBidBillable](/dev-docs/bidder-adaptor.html#registering-on-bid-billable) for more info. |
+| `bidLimit` | Optional | Number | Used by a publisher to set a bid limit for this ad unit |
```

Comment:
Could you add a small, separate section and explain what a `bidLimit` implies? 
Is the for the entire page session, are all bidders affected, what's the main use case?